src/prpy/util.py

```diff
@@ -959,6 +959,42 @@ def CheckJointLimits(robot, q):
             description='position')
 
 
+def GetForwardKinematics(robot, q, manipulator=None, frame=None):
```

Comment:
An API suggestion: remove the `robot` parameter and just take `manipulator`. You can get the robot from `manipulator.GetRobot()`. Thoughts?